templates/.circleci/config.yml.tpl

```diff
@@ -124,7 +124,7 @@ workflows:
     when: << pipeline.parameters.rebuild_cache >>
     jobs:
       - shared/save_cache: *test
-  {{- if stencil.Arg "releaseOptions.autoPrereleases" }}
+  {{- if and (stencil.Arg "releaseOptions.enablePrereleases") (stencil.Arg "releaseOptions.autoPrereleases") }}
```

Comment:
Added this check to ensure if `enablePrereleases` is disabled, the autoPrereleases will be ignored.

Reply:
Do we need extra tests for this now that there's two options needed?

Reply:
I briefly tested it but the two options are mutual. I will add the test case, and comment it as only to verify the corner case.